### DISCUSSION TITLE
ToolJet

### DISCUSSION THREAD

**Discussion by danielehrhardt** [coollabsio/coolify#4020] [Service Template Requests]
https://github.com/ToolJet/ToolJet?tab=readme-ov-file#quickstart

**Comment by SpuriousGer** [comment]
It would be cool if this was available as a one-click deployment.

**Reply by Recreabox** [reply]
> It would be cool if this was available as a one-click deployment.

Yes it would be so useful, please can you add Tooljet ?

**Comment by Nirator78** [comment]
Any update for tooljet integration ?

**Comment by stockarea** [comment]
Are we working on this?
